.circleci/getChangelogs.sh

```diff
@@ -32,23 +32,46 @@ function cloneRepository() {
     old_version=$2
     new_version=$3
     repository_url=$4
-    echo "[OK] $project_name($repository_url) : $old_version -> $new_version"
+    echo "[OK] checkout: $project_name($repository_url)"
     git clone "$repository_url" "$project_name" --quiet
 }
 
+function getOldDependencyVersion(){
```

Comment:
If there is a new component introduced in the requirements file, it will not have an old version.

this has happened with codacy-ingress in the release it was introduced, where the changelog had no info about it.